db/migrate/20151125004633_create_pages.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePages < ActiveRecord::Migration
```

Comment:
@cawel Would you be up to make title and body "globalized"? If not happy to accept the PR as is.

https://github.com/globalize/globalize#creating-translation-tables